src/Grpc.Net.ClientFactory/GrpcHttpClientBuilderExtensions.cs

```diff
@@ -379,7 +379,7 @@ public static IHttpClientBuilder ConfigureGrpcClientCreator(this IHttpClientBuil
         return builder;
     }
 
-    private static void ValidateGrpcClient(IHttpClientBuilder builder)
+    private static void ValidateGrpcClient(IHttpClientBuilder builder, string caller)
```

Comment:
would this be cleaner as `[CallerMemberName] string caller = null!` ?

Reply:
I didn’t because there aren’t tests for each name, and someone could unknowingly move this method to a child method and have the wrong name.